packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -121,7 +121,7 @@ export default class QuanticResultLink extends NavigationMixin(
     this.engine = engine;
     ResultUtils.bindClickEventsOnResult(
       this.engine,
-      this.result,
+      {...this.result, raw: {...this.result.raw}},
```

Comment:
Why do we need to de-structure the object here?

Reply:
Because Headless is not dealing correctly with the Proxy  object.
As we know Salesforce converts object to Proxy objects, so with this procedure, the de-structuring, I'm converting the Proxy object back to a normal object.

Reply:
And it works by de-structuring? You were able to reproduce before and can no longer reproduce after?

Reply:
Exactly like you can see in the recordings included in the PR description the error we are trying to solve is the `TypeError isExtensible....`

That's an example of doing console.log(this.result) amd console.log({...this.result, raw: {...this.result.raw}}):
<img width="748" alt="Screenshot 2024-06-12 at 9 48 00 AM" src="https://github.com/coveo/ui-kit/assets/86681870/5564dd58-c5d2-4ddb-8f7c-6b0aae3da1dc">